dask_expr/_shuffle.py

```diff
@@ -615,12 +619,23 @@ class AssignPartitioningIndex(Blockwise):
         Number of partitions after repartitioning is finished.
     """
 
-    _parameters = ["frame", "partitioning_index", "index_name", "npartitions_out"]
+    _parameters = [
+        "frame",
+        "partitioning_index",
+        "index_name",
+        "npartitions_out",
+        "assign_index",
```

Comment:
Can we add this to the doc-string (I think the name is fine, but is still a bit vague)?

Reply:
yep